dali/pipeline/pipeline.cc

```diff
@@ -518,21 +468,6 @@ void Pipeline::Build(std::vector<PipelineOutputDesc> output_descs) {
   executor_->EnableCheckpointing(checkpointing_);
   executor_->Init();
 
-  // Creating the graph
-
-  for (auto& name_op_spec : op_specs_) {
-    string& inst_name = name_op_spec.instance_name;
-    OpSpec op_spec = name_op_spec.spec;
-    PrepareOpSpec(&op_spec, name_op_spec.logical_id);
-    try {
-      graph_builder_.Add(inst_name, op_spec);
-    } catch (...) {
-      PropagateError({std::current_exception(),
-                      "Critical error when building pipeline:\n" + GetErrorContextMessage(op_spec),
-                      "\nCurrent pipeline object is no longer valid."});
-    }
-  }
-
```

Comment:
This part has been moved _after_ the outputs because when processing outputs we add MakeContiguous operators.